tests/fixtures/InitCommandTest/auto-doc.php

```diff
@@ -0,0 +1,184 @@
+<?php
```

Comment:
after mocking get config content - let's simplify it to small array with few keys